src/main/java/com/erstudio/ERSStudioReleaseHelperUI.java

```diff
@@ -423,9 +424,20 @@ public void onPressedReset() {
             buttonGroupUpgradeTypeTS.clearSelection();
             textGUIDTS.setText("");
             textGUIDTS.setBackground(Color.WHITE);
+            textAreaStatusTS.setText("");
         }
         if (tabbedPaneUI.getTitleAt(tabbedPaneUI.getSelectedIndex()) == Constants.DATA_ARCHITECT) {
-
```

Comment:
Why this has been removed?

Reply:
Vaibhav i have added the code for reset button for DA so I have added code here